Vostok.Applications.AspNetCore/Middlewares/PingApiMiddleware.cs

```diff
@@ -55,9 +62,9 @@ private Task HandleVersionRequest(HttpContext context)
         }
 
         private string GetHealthStatus()
-            => initialized.Value ? settings.HealthCheck?.Invoke() ?? true ? "Ok" : "Warn" : "Init";
+            => options.Value.InitializationCheck?.Invoke() ?? true ? options.Value.HealthCheck?.Invoke() ?? true ? "Ok" : "Warn" : "Init";
```

Comment:
Looks complicated — 8 `?` symbols in a single line & nested ternary operator. IMO 'if' version should be preferred for outer condition.

Reply:
Yup, agreed. Fixed it.